src/platform-includes/getting-started-config/apple.mdx

```diff
@@ -42,6 +46,10 @@ struct SwiftUIApp: App {
         SentrySDK.start { options in
             options.dsn = "___PUBLIC_DSN___"
             options.debug = true // Enabled debug when first installing is always helpful
+
+            // Enable tracing to capture 100% of transactions for performance monitoring.
+            // If you want to choose the sampling rate, you can use 'options.tracesSampleRate'
```

Comment:
// Enable tracing to capture 100% of transactions for performance monitoring.
            // Use 'options.tracesSampleRate' to set the sampling rate. 
            // We recommend setting a sample rate in production.
```